.npmrc

```diff
@@ -0,0 +1,5 @@
+# super strict mode
+auto-install-peers=false
```

Comment:
`strict-peer-dependencies=true` might also be useful to you

Reply:
ooo good idea!